app/src/main/java/com/li/weatherapp/ui/splash/Splash.kt

```diff
@@ -0,0 +1,26 @@
+package com.li.weatherapp.ui.splash
+
+import android.os.Handler
+import com.li.weatherapp.R
+import com.li.weatherapp.base.BaseActivity
+import com.li.weatherapp.ui.main.MainActivity
+
+class Splash : BaseActivity() {
+    override val layoutResource: Int
+        get() = R.layout.activity_splash
+
+    override fun initViews() {
+
+    }
+
+    override fun initActions() {
+        Handler().postDelayed({
+            startActivity(MainActivity.getIntent(this))
+            finish()
+        }, TIME)
+    }
```

Comment:
Áp dụng được không ? 
https://medium.com/android-news/right-way-to-create-splash-screen-on-android-e7f1709ba154